ios/RNSScreen.mm

```diff
@@ -664,11 +1110,6 @@ - (BOOL)hasHeaderConfig
   return _config != nil;
 }
 
-+ (react::ComponentDescriptorProvider)componentDescriptorProvider
-{
-  return react::concreteComponentDescriptorProvider<react::RNSScreenComponentDescriptor>();
-}
-
```

Comment:
Why?

Reply:
Had some issues with Fabric compilation, obviously will restore it 😅